### PR TITLE
dev-python/imageio: new ebuild

### PR DESCRIPTION
**Pull request by TheChymera**
Imageio is a pure-python library with few deps (only numpy) that provides read/write functionality for very many formats and which is easily extensible. 

Maintainership request submitted: https://bugs.gentoo.org/show_bug.cgi?id=592384

### CODE REVIEW

File: dev-python/imageio/ChangeLog [hunk 1]
```diff
@@ -0,0 +1,5 @@
+*imageio-1.5 (29 Aug 2016)
```
Comment (SoapGentoo) [inline dev-python/imageio/ChangeLog]: remvoe the `ChangeLog` file, these are autogenerated if need be

File: dev-python/imageio/imageio-1.5.ebuild [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_5 )
```
Comment (SoapGentoo) [inline dev-python/imageio/imageio-1.5.ebuild]: maybe also add py3.4? Just a suggestion, your call

File: dev-python/imageio/imageio-1.5.ebuild [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+SRC_URI="https://github.com/imageio/imageio/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~x86 ~amd64"
+IUSE=""
+
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	"
```
Comment (SoapGentoo) [inline dev-python/imageio/imageio-1.5.ebuild]: make `DEPEND` and `RDEPEND` on one line each, don't need to blow it up unnecessarily.

File: dev-python/imageio/imageio-1.5.ebuild [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+
+inherit distutils-r1
+
+DESCRIPTION="Video editing with Python"
```
Comment (radhermit) [inline dev-python/imageio/imageio-1.5.ebuild]: Shouldn't DESCRIPTION be "a Python library for reading and writing image data" or something similar? I assume you copy/pasted the ebuild from moviepy and forgot to update it.

File: dev-python/imageio/imageio-1.5.ebuild [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+HOMEPAGE="https://imageio.github.io/"
+SRC_URI="https://github.com/imageio/imageio/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~x86 ~amd64"
+IUSE=""
+
+RDEPEND="dev-python/numpy[${PYTHON_USEDEP}]"
+DEPEND="
```
Comment (SoapGentoo) [inline dev-python/imageio/imageio-1.5.ebuild]: I know, pedantic, but please change this to 

```
DEPEND="${RDEPEND}
	dev-python/setuptools[${PYTHON_USEDEP}]"
```

**Comment by SoapGentoo** [comment]
@TheChymera what about the freeimage thing? Can we add that as an `RDEPEND`, seeing that it is pretty much required (raising an **ImportError** exception is not what Gentoo considers "optional dependencies").

**Comment by TheChymera** [comment]
@SoapGentoo Addressed.

**Pull request closed without merging**